### DISCUSSION TITLE
Cant run pybamm examples in google colab

### DISCUSSION THREAD

**Discussion by sreehari33** [pybamm-team/PyBaMM#2532] [Q&A]
I've been trying to run some examples in google colab, but while trying to install pybamm I am getting this error:
PluginNotFound: plugin pybtex.database.input.bibtex not found
please help.

**Comment by valentinsulzer** [comment]
https://github.com/pybamm-team/PyBaMM/issues/2524#issuecomment-1338818127

**Comment by DrSOKane** [comment]
Which examples? I've also had issues, but only with some parameter sets. Others have run fine.
